src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/BooleanExpressionComplexity/context/defaultContext.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/Test.java b/Test.java
+index 7ad95ed..98f9127 100644
+--- a/Test.java
++++ b/Test.java
+@@ -4,9 +4,9 @@ public class Test {
+     public void test(boolean a, boolean b) {
+         boolean c = (a & b) | (b ^ a);
+ 
+-        boolean d = (a & b) ^ (a || b) | b;  // violation without filter
```

Comment:
Please don't change this line for context strategy. Thus, a violation is present without filter but is not expected in the context strategy.

Reply:
done